pyproject.toml

```diff
@@ -37,7 +37,7 @@ Repository = "https://github.com/openai/openai-agents-python"
 [project.optional-dependencies]
 voice = ["numpy>=2.2.0, <3; python_version>='3.10'", "websockets>=15.0, <16"]
 viz = ["graphviz>=0.17"]
-litellm = ["litellm>=1.67.4.post1, <2"]
+litellm = ["litellm>=1.80.8, <2"]
```

Comment:
Is this upgrade required for this PR? If not, can you move this change and uv.lock changes to a different PR? Having uv.lock changes could block merging this PR due to non-essential conflicts

Reply:
okay, thanks